freeipa/src/main/java/com/sequenceiq/freeipa/service/recipe/RecipeUsageService.java

```diff
@@ -0,0 +1,54 @@
+package com.sequenceiq.freeipa.service.recipe;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import com.cloudera.thunderhead.service.common.usage.UsageProto;
+import com.sequenceiq.cloudbreak.usage.UsageReporter;
+
+@Service
+public class RecipeUsageService {
```

Comment:
Why is this Service duplicated?

Reply:
the core one is based on Recipe entity, this one is based on limited availability of information regarding recipe, but of course I can make a common class with common parts